<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1512</width>
    <height>888</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Введите смещение dx, dy, dz</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="dxCarry"/>
      </item>
      <item>
       <widget class="QLineEdit" name="dyCarry"/>
      </item>
      <item>
       <widget class="QLineEdit" name="dzCarry"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Scale">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Масштабирование</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Введите коэффициенты kx, ky, kz</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="kxScale"/>
      </item>
      <item>
       <widget class="QLineEdit" name="kyScale"/>
      </item>
      <item>
       <widget class="QLineEdit" name="kzScale"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Введите угол поворота(в градусах)</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLineEdit" name="angleRotate"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Carryover">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Перенос</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>291</x>
     <y>11</y>
     <width>1161</width>
     <height>851</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Rotate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>810</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Поворот</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>1161</width>
     <height>851</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10"/>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Введите точку </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>центра масштабирования x, y, z</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="xScale"/>
      </item>
      <item>
       <widget class="QLineEdit" name="yScale"/>
      </item>
      <item>
       <widget class="QLineEdit" name="zScale"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>710</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Введите центр поворота x, y, z</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLineEdit" name="xRotate"/>
      </item>
      <item>
       <widget class="QLineEdit" name="yRotate"/>
      </item>
      <item>
       <widget class="QLineEdit" name="zRotate"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="LoadFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Загрузить файл</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
